src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -3105,6 +3105,8 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST(GUITest_common_scenarios_NIAID_pipelines::test_0002);
     REGISTER_TEST_TL(GUITest_common_scenarios_NIAID_pipelines::test_0003, minutes(20), labels({Cistrome, Linux}));
     REGISTER_TEST_TL(GUITest_common_scenarios_NIAID_pipelines::test_0004, minutes(20), labels({Cistrome, Linux}));
+    REGISTER_TEST(GUITest_common_scenarios_NIAID_pipelines::test_0005, minutes(7));
```

Comment:
Tests which are longer than the default timeout (today it is 4 minutes) are not allowed in the Nightly test suite.

Please move such tests into a special Cistrome suite: just add labels({Cistrome, Linux}) to the tests registration.